src/app/routes/storage.js

```diff
@@ -244,12 +244,24 @@ module.exports = (Router, Service, App) => {
       });
   });
 
+  Router.post('/storage/folder/fixduplicate', passportAuth, (req, res) => {
```

Comment:
This endpoint could be removed if @AlexMenor confirms that desktop is not using it, part of the legacy old desktop code.

Reply:
its not, but keep in mind that not everyone is on > 1.5.0 yet

Reply:
Ok, i wont remove it yet. In any case we can open another PR for cleaning purposes.